cmd/influx/write_test.go

```diff
@@ -0,0 +1,414 @@
+package main
```

Comment:
I introduced this test, because I wanted to keep the playground cleaner than before I entered, but it might not look so, ok :) So I will write down more comments herein to know what is being tested.